docs/astrocut/index.rst

```diff
@@ -142,45 +142,51 @@ treated as the R, G, and B channels respectively.
+(this only needs to be completed once per set of FFIs), and to then make individual cutout TPFs from this
+large cube file for targets of interest. If you are creating a small number of cutouts, the TESSCut web service
+may suit your needs: `mast.stsci.edu/tesscut <https://mast.stsci.edu/tesscut/>`_
  
 Making image cubes
 ------------------
 
-Making an image cube is a simple operation, but comes with an important
-time/memory trade-off.
-
 .. important::
-   **Time/Memory Trade-off**
+   **Time-Memory Trade-off**
+
+   Making an image cube is a simple operation, but comes with an important
+   time-memory trade-off.
 
    The ``max_memory`` argument determines the maximum memory in GB that will be used
-   for the image data cube while it is being built. This is *only* for the data cube,
-   and so is somewhat smaller than the amount of memory needed for the program to run.
-   Never set it to your system's total memory.
-
-   Because of this, it is possible to build cube files with much less memory than will
-   hold the final product. However there is a large time trade-off, as the software must
-   run through the list of files multiple times instead of just once. The default value
-   of 50 GB was chosen because it comfortably fits a main mission sector of TESS FFIs,
-   with the default setting on a system with 65 GB of memory it takes about 15 min to
-   build a cube file. On a system with enough less memory that 3 passes through the
-   list of files are required this time rises to ~45 min. 
+   for the image data cube while it is being built. This is the amount of memory required 
+   *only* for the data cube, so is somewhat smaller than the total amount of memory needed
+   for the program to run. You should never set it to your system's total memory.
+
+   Because of this, cube files do not need to allocate their total size in
+   memory all at once. Instead, a smaller memory allocation can be used while
+   the cube file is constructed; however, this will significantly increase the
+   execution time as bytes are swapped into and out of the memory allocation 
+   being used. The default value of 50 GB was chosen because it fits all of the
+   TESS FFIs from a single Prime Mission Sector (Sectors 1-26); with the
+   default settings, on a system with 65 GB of memory, it takes about 15 min to
+   build a single cube file. On a system with less memory, where e.g., 3
+   passes through the list of files are required, this time increases to 
+   approximately 45 min.
    
 
-By default `~astrocut.CubeFactory.make_cube` runs in verbose mode and prints out its progress, however setting
-verbose to false will silence all output.
+Assuming that you have set of calibrated TESS (or TICA) FFI files stored locally, you can
+create a cube using the `~astrocut.CubeFactory.make_cube` method (or 
+`~astrocut.TicaCubeFactory.make_cube` for TICA products). By default, `~astrocut.CubeFactory.make_cube` 
```

Comment:
Both `CubeFactory.make_cube` and `TicaCubeFactory.make_cube` run with `verbose=True` by default